doc/community-build/Generate-Cortx-Build-Stack.md

```diff
@@ -66,87 +59,110 @@ To know about various CORTX components, see [CORTX Components guide](https://git
+   ```
 
-      cortx-ha: generate cortx-ha packages.
+   **Note:** It may take more than an hour to generate all the CORTX packages.
 
-      cortx-management-portal: generate cortx-management-portal packages.
+12. Run the following command to generate the ISO for each component:
 
-      cortx-manager: generate cortx-manager packages.
+   ```
+   docker run --rm -v /var/artifacts:/var/artifacts -v /root/cortx:/cortx-workspace ghcr.io/seagate/cortx-build:centos-7.9.2009 make iso_generation
+   ```
 
-      cortx-monitor: generate cortx-monitor packages.
+13. The CORTX build is generated in the directory created at step 3. To view the generated build, run:
 
-      cortx-posix: generate cortx-posix (NFS) packages.
+   ```
+   [root@deploy-test ~]# ll /var/artifacts/0/
+total 1701408
+drwxr-xr-x  13 root root        231 Aug 30 21:44 3rd_party
+drwxr-xr-x   3 root root       4096 Aug 30 21:44 cortx_iso
+-rw-r--r--   1 root root      19416 Aug 30 21:44 install-2.0.0-0.sh
+drwxr-xr-x   2 root root         38 Aug 30 22:44 iso
+drwxr-xr-x 198 root root       8192 Aug 30 21:44 python_deps
+-rw-r--r--   1 root root  241635505 Jun 15 14:45 python-deps-1.0.0-0.tar.gz
+-rw-r--r--   1 root root 1500564340 Jul 22 12:16 third-party-1.0.0-0.tar.gz
+   ```
 
-      cortx-prvsnr: generate cortx-prvsnr packages.
+14. To view each component targets, run:
 
-      iso_generation: generate ISO file from release build.
    ```
+   [root@deploy-test ~]# docker run ghcr.io/seagate/cortx-build:centos-7.9.2009 make help
+usage: make "target"
 
-8. Deploy the packages generated to create CORTX cluster using the instruction provided in [Deploy Cortx Build Stack guide](ProvisionReleaseBuild.md).
-
+Please clone required component repositories in cortx-workspace directory before executing respective targets.
 
-## Troubleshooting
+targets:
+    help: print this help message.
+    checkout: checkout specified branch/tag code
+    clean: remove existing /var/artifacts/0 directory.
+    build: generate complete CORTX build including third-party-deps at "/var/artifacts/0"
+    control-path: generate control-path packages. cortx-provisioner, cortx-monitor, cortx-manager, cortx-management-portal and cortx-ha.
+    io-path: generate io-path packages. cortx-motr, cortx-s3server and cortx-hare.
+    cortx-motr: generate cortx-motr packages.
+    cortx-s3server: generate cortx-s3server packages.
+    cortx-hare: generate cortx-hare packages.
+    cortx-ha: generate cortx-ha packages.
+    cortx-management-portal: generate cortx-management-portal packages.
+    cortx-manager: generate cortx-manager packages.
+    cortx-monitor: generate cortx-monitor packages.
+    cortx-posix: generate cortx-posix (NFS) packages.
+    cortx-prvsnr: generate cortx-prvsnr packages.
+    iso_generation: generate ISO file from release build.
+    ```   
 
-You might get an error message about missing `kernel-devel` package when building the CORTX packages; sample errors:
```

Comment:
I feel it's good to have a troubleshooting section. Any specific reason to remove it.